main.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <string.h>
+
+
+
+
+
+void swap(char* a1, char* a2)
+{
+  char t = *a1;
+  *a1 = *a2;
+  *a2 = t;
+}
+
+void reverse(char* A, int size)
+{
+  char* a1 = A;
+  char* a2 = A + size - 1;
+  while (a1 < a2)
+  {
+    swap(a1, a2);
+    a1++;
+    a2--;
+  }
+}
+
+int main()
+{
+    while(1)
```

Comment:
An endless cycle, why need it here I don't understand. Add a comment for purpose you created this